arches/app/media/js/views/components/cards/photo-gallery-card.js

```diff
@@ -49,6 +49,16 @@ define([
 
         this.fileListNodeId = getfileListNode();
 
+        this.maxFilesize = ko.computed(function(){
+            var mfs = "Missing maxFilesize";
+            self.card.widgets().forEach(function(widget){
+                if (widget.node_id() === self.fileListNodeId) {
+                    mfs = widget.config.maxFilesize();
```

Comment:
```suggestion
                    mfs = widget.config.maxFilesize() || "undefined";
```

Quite so! If undefined, let's inform the user. ^_^

![image](https://github.com/archesproject/arches/assets/53860784/e308a56e-6a75-4bd0-a0c7-091987699e4c)

Reply:
Something symbolic like `--` might be even better.

Reply:
Maybe that entire sentence doesn't need to be displayed if there is no max size?

Reply:
I think we should go with @jacobtylerwalls `--` suggestion. That way we can keep the message and conveys to the user that no max is enforced.